rust/basic_bitcoin/dfx.json

```
@@ -0,0 +1,37 @@
+{
+  "version": 1,
+  "dfx": "0.11.0",
+  "canisters": {
+    "basic_bitcoin": {
+      "type": "custom",
+      "package": "basic_bitcoin",
+      "candid": "src/basic_bitcoin/basic_bitcoin.did",
+      "wasm": "target/wasm32-unknown-unknown/release/basic_bitcoin.wasm",
+      "build": "src/basic_bitcoin/build.sh"
+    },
+    "ecdsa": {
+      "type": "custom",
+      "package": "ecdsa",
+      "candid": "ecdsa.did",
+      "wasm": "ecdsa.wasm",
+      "build": ""
```

```suggestion
      "build": []
```

Makes the `Executing ''` goes away when running `dfx deploy`.